### DISCUSSION TITLE
https://y9san9.me/blog/West-Approach-to-Russia-was-a-Mistake

### DISCUSSION THREAD

**Discussion by giscus** [y9san9/public-vault#21] [Announcements]
# https://y9san9.me/blog/West-Approach-to-Russia-was-a-Mistake

Media hop on the trend and start fighting against Russia as well, but this time in information warfare. This is where Western strategy went wrong.

https://y9san9.me/blog/West-Approach-to-Russia-was-a-Mistake

**Comment by demndevel** [comment]
literally what's in my head

**Comment by portasynthinca3** [comment]
Great post. I completely agree. Thank you for putting my thoughts into words.

**Comment by kr0fff** [comment]
"You can find several people who do believe in TV, but it’s much less than you might expect"

Sorry, but I think you are just living in your own info-bubble, there are still a lot of "ZOV" labels in telegram nicknames and military contracts are only increasing

**Reply by y9san9** [reply]
Yep, this is true. Yet those people don't think that someone on TV tells them the truth. Those proactive people believe in some abstract goal, not in what TV says to them.
